zerver/models.py

```diff
@@ -551,7 +551,7 @@ class UserProfile(ModelReprMixin, AbstractBaseUser, PermissionsMixin):
     is_active = models.BooleanField(default=True, db_index=True) # type: bool
     is_realm_admin = models.BooleanField(default=False, db_index=True) # type: bool
     is_bot = models.BooleanField(default=False, db_index=True) # type: bool
-    bot_type = models.PositiveSmallIntegerField(null=True, db_index=True) # type: Optional[int]
+    bot_type = models.PositiveSmallIntegerField(default=1, null=True, db_index=True) # type: Optional[int]
```

Comment:
This change is wrong -- it'll give `bot_type` values to non-bot users.  You need to change the code that creates bot users, not this.